starter_templates/redis/rust/app/main.rs

```diff
@@ -0,0 +1,15 @@
+
+
+use std::net::TcpListener;
+
+fn main() {
+
+    println!("Your code goes here!")
+
+     // Uncomment the below lines to pass first stage
+  
+      //TcpListener::bind("127.0.0.1:6379").unwrap(); 
```

Comment:
The indentation/spacing looks inconsistent here - I'm seeing 4/5/6 spaces in different lines. Please use a code formatter to ensure this follows standard formatting rules.

Reply:
Acknowledged sir